streams/src/test/java/org/apache/kafka/streams/processor/internals/TaskManagerTest.java

```diff
@@ -2470,7 +2474,7 @@ public void markChangelogAsCorrupted(final Collection<TopicPartition> partitions
 
         assertThat(unrevokedTaskChangelogMarkedAsCorrupted.get(), is(true));
         assertThat(revokedActiveTask.state(), is(State.SUSPENDED));
-        assertThat(unrevokedActiveTask.state(), is(State.CREATED));
+        assertThat(unrevokedActiveTask.state(), is(State.RUNNING));
```

Comment:
@ableegoldman , this test has undergone some change. Most of it seems cosmetic to me but this one is a deviation from what the original test case was. This change seemed reasonable to me. Let me know what you think about it.